contributing.md

```diff
@@ -1,10 +1,10 @@
 ## Contributing
 
-We welcome contributions!
+We welcome [contributions](https://help.github.com/articles/about-pull-requests/)!
 
 We're using Github to write this guide as it's more likely to persist longer and allows for other people to discuss and contribute changes. It's a relatively new style of writing that most of us don't have experience with, so we're all experimenting on this together!
 
-A few tips on making good PRs:
+A few tips on making [good PRs](https://github.com/blog/1943-how-to-write-the-perfect-pull-request):
```

Comment:
yeah the reason I wanted you to link to it was because I thought people would be confused about what PR was (that it stands for pull request, and is a way of making changes to the repo)